flake8_isort.py

```diff
@@ -1,12 +1,10 @@
 # -*- coding: utf-8 -*-
 from difflib import Differ
-from flake8_polyfill import stdin
```

Comment:
If this is not needed anymore, we could remove it from setup.py. Would you mind doing it? TIA